front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -22,6 +25,35 @@ const ScenarioDescription = ({
 }: ScenarioDescriptionProps) => {
   const { t } = useTranslation('operationalStudies/scenario');
   const { openModal } = useModal();
+  const [isOpenedDescription, setIsOpenedDescription] = useState<boolean>(false);
+  const descriptionRef = useRef<HTMLDivElement | null>(null);
+  const ref = useRef<HTMLDivElement | null>(null);
```

Comment:
`ref` is a bit vague. Maybe rename to `collapsedDescriptionRef`?

Reply:
Done